frame/executive/src/lib.rs

```diff
@@ -383,7 +363,9 @@ where
 {
 	/// Execute all `OnRuntimeUpgrade` of this runtime, and return the aggregate weight.
 	pub fn execute_on_runtime_upgrade() -> Weight {
-		<(COnRuntimeUpgrade, AllPalletsWithSystem) as OnRuntimeUpgrade>::on_runtime_upgrade()
+		<(COnRuntimeUpgrade, AllPalletsWithSystem) as frame_support::traits::Hooks<
+			System::BlockNumber,
+		>>::on_runtime_upgrade()
```

Comment:
this is different than before because `OnRuntimeUpgrade` implementation is usually different than the hook implementation